### PR TITLE
feat(healthcheck.go): add Ping and HealthCheck routes

### PR DESCRIPTION
**Pull request by MateSousa**
## Description

The Ping function responds with "pong" and returns the latency. The HealthCheck function checks the server and database status. If the database connection fails, it returns a 500 status code. If the ping to the database fails, it also returns a 500 status code. Otherwise, it returns a 200 status code and "OK".

## Related issue(s)

#111

### CODE REVIEW

File: controllers/healthcheck.go
Comment (harshkhandeparkar) [inline controllers/healthcheck.go:42]: Can you also add a check to handle this error?

File: server/routes.go
Comment (harshkhandeparkar) [inline server/routes.go:25]: Since strict slashes are enabled, can you renamed this path to `/healthcheck/`. Also there is no need to add `/api/`

File: server/routes.go
Comment (harshkhandeparkar) [inline server/routes.go:31]: Here too, `/api/` can be removed.

**Review by harshkhandeparkar — CHANGES_REQUESTED** [review]

**Comment by harshkhandeparkar** [comment]
Thank you for the contribution! We'll merge this once #108 is merged.

**Comment by rajivharlalka** [comment]
LGTM, Thanks a lot for the PR @MateSousa :)

**Pull request merged**
